mdnsbridge/mdnsbridgeclient.py

```diff
@@ -76,28 +76,50 @@ def getHref(self, srv_type, priority=None, api_ver=None, api_proto=None):
                 continue
             if api_proto is not None and api_proto != service["protocol"]:
                 continue
+
             if priority >= 100:
                 if service["priority"] == priority:
-                    return self._createHref(service)
+                    return [service]
```

Comment:
Was the intention not to gather the full list of services matching this fixed priority?